packages/core/src/State.ts

```diff
@@ -85,8 +112,8 @@ interface MachineSnapshotBase<
   children: TChildren;
 
   /**
-   * Whether the current state value is a subset of the given parent state value.
-   * @param testValue
+   * Whether the current state value is a subset of the given partial state value.
+   * @param partialStateValue
```

Comment:
partial sounds like literally a partial of the state value but for a state value like `{ a: 'b' }` we can also test against `'a'`

Reply:
Yeah, was just trying to figure out something less ambiguous than `testValue`, which doesn't suggest a state value